test/image/mocks/pattern_bars.json

```diff
@@ -56,7 +56,27 @@
     }
    }
   },
-
+  {
+    "x": ["a", "b", "c", "d", "e"],
+    "y": [7, 8, 9, 10, 11],
+    "name": "Bar 5",
+    "type": "bar",
+    "textposition": "outside",
+    "text": "path",
+    "marker": {
+     "pattern": {
+      "path": [
+        "M0,0H4V4H0Z",
+        "M0,0H6V6Z",
+        "M0,0V4H4Z",
+        "M0,0C0,2,4,2,4,4C4,6,0,6,0,8H2C2,6,6,6,6,4C6,2,2,2,2,0Z",
+        "M4,4L7,2A3.5,3.5,0,1,0,7,6Z"
+      ],
+      "fgcolor": "yellow",
+      "bgcolor": "black"
+     }
+    }
+   },
```

Comment:
I'd like to keep `pattern_bars.json` as it was before.
Please duplicate the mock and name the new mock `zz-pattern_bars-path.json`.

This is wonderful feature.
Thanks @alexcjohnson for the PR. :trophy: 

We may introduce new features like this one in `v3.1.0` not `v3.0.0`; even though one may argue that it's a low risk feature. BTW it requires extra testing on the python side, etc. 
cc: @gvwilson

Reply:
OK - personally I think the mocks are getting a little excessive, and we'd be better off testing as many things as we can in one mock, which is why I just added this to one of the existing mocks. But I've made the change.